tools/ref-tag/create.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+MY_CMD="${0}"
+MY_CMD_ABS="$(readlink -e "${MY_CMD}")"
```

Comment:
I will modify the script in a branch, that you could test locally on your MAC. Okay? I tested with linux only.